docs/source/api/algorithm_functions/connectivity_and_cycles.rst

```diff
@@ -25,3 +25,6 @@ Connectivity and Cycles
    rustworkx.all_pairs_all_simple_paths
    rustworkx.stoer_wagner_min_cut
    rustworkx.longest_simple_path
+   rustworkx.is_bipartite
+   rustworkx.isolates
+   rustworkx.has_path
```

Comment:
I will try to catch the documentation entries more often, this release we let many slip through